cmd/run-function/runfn/runfn.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package runfn
+//Reference: sigs.k8s.io/kustomize/kyaml/runfn
```

Comment:
I may be missing it. But this looks like something we could just include and use as a dependency.

https://pkg.go.dev/sigs.k8s.io/kustomize/kyaml@v0.13.7/runfn

Reply:
Like I said in [the comment below](https://github.com/konveyor/crane/pull/125#discussion_r912159637), my personal preference would be to import modules wherever possible. If you were to create a fork, then a couple of things:
1. If they make changes we can bring those into the fork with `rebase`.
2. We can submit the work needed to support `podman` "upstream", be good community citizens, and all that jazz.